ReactiveCocoa/Swift/Signal.swift

```diff
@@ -1216,6 +1216,57 @@ extension SignalType {
 	}
 }
 
+extension SignalType {
+	/// Forwards only those values from `self` that have unique identities across the set of
+	/// all values that have been seen.
+	///
+	/// Note: This causes the identities to be retained to check for uniqueness.
+	@warn_unused_result(message="Did you forget to call `observe` on the signal?")
+	public func uniqueValues<Identity: Hashable>(transform: Value -> Identity) -> Signal<Value, Error> {
+		return Signal { observer in
+			let seenValues: Atomic<Set<Identity>> = Atomic([])
```

Comment:
Is the `Atomic` use really needed? It looks like the set is not accessed or modified concurrently in the implementation.